File: UnitTests/BraintreeCoreTests/BTCacheDateValidator_Tests.m
#import <XCTest/XCTest.h>
#import "BTCacheDateValidator_Internal.h"

@interface BTCacheDateValidator_Tests : XCTestCase

@end

@implementation BTCacheDateValidator_Tests

- (void)testTimeToLiveMinutes_defaultsTo5 {
    BTCacheDateValidator *cacheDateValidator = [[BTCacheDateValidator alloc] init];
    XCTAssertEqual(5, cacheDateValidator.timeToLiveMinutes);
}

@end

File: UnitTests/BraintreeCoreTests/BTCacheDateValidator_Tests.swift
import XCTest
@testable import BraintreeCore

class BTCacheDateValidator_Tests: XCTestCase {
    func testTimeToLiveMinutes_defaultsTo5() {
        let cacheDateValidator = BTCacheDateValidator()
        XCTAssertEqual(5, cacheDateValidator.timeToLiveMinutes)
    }
}
